packages/core/src/ceramic.ts

```diff
@@ -632,7 +648,10 @@ export class Ceramic implements CeramicApi {
     const walkNext = async (node: TrieNode, streamId: StreamID | CommitID) => {
```

Comment:
I think all you need to do is put the genesis contents into IPFS before the call to walkNext